# Return all static obstacles in one section
uint8 id
---
LabeledPolygon[] polygons
